front/src/views/recovery-password/RecoveryPassword.html

```diff
@@ -0,0 +1,162 @@
+      color="primary lighten-1"
+    >
+      <h4>{{$t('recoverypassword.title')}}</h4>
+    </v-card-title>
+  </v-row>
+  <v-card-text>
+    <v-stepper non-linear alt-labels elevation="0" v-model="step">
+      <v-stepper-header>
+        <v-stepper-step :complete="step > 1" step="1">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.checkMail')}}
+          </div>
+        </v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step :complete="step > 2" step="2">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.title')}}
+          </div>
+        </v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step step="3">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.Conf')}}
+          </div>
+        </v-stepper-step>
+      </v-stepper-header>
+      <v-stepper-items>
+        <v-stepper-content step="1">
+          <v-card color="#F6F8FB">
+            <div class="text-center text-normal">
+              {{$t('recoverypassword.Entuemail')}}
+            </div>
+            <v-form ref="form" v-model="valid">
+              <div class="input-email">
+                <mmt-text-input
+                  class="input-field"
+                  mmt-label="E-mail"
+                  rounded
+                  filled
+                  single-line
+                  dense
+                  :placeholder="$t('recoverypassword.emailPlh')"
+                  v-model="email"
+                  :rules="[rules.isEmpty, rules.validEmail]"
+                  :error-messages="serverError"
+                  @input="onInput"
+                  required
+                />
+              </div>
+              <v-row cols="2" justify="center">
+                <v-col justify="center">
+                  <v-btn
+                    class="mmt-btn btn-step-1"
+                    rounded
+                    color="primary"
+                    :disabled="!valid"
+                    @click="recoveryPassword"
+                    >{{$t('recoverypassword.recover')}}
+                  </v-btn>
+                </v-col>
+                <v-col justify="center">
+                  <v-btn
+                    rounded
+                    outlined
+                    color="accent lighten-1"
+                    class="mmt-btn btn-step-1"
+                    dense
+                    @click="$router.push('/');"
+                    >{{$t('recoverypassword.back')}}
+                  </v-btn>
+                </v-col>
+              </v-row>
+            </v-form>
+            <div class="text-center text-normal">
+              {{$t('recoverypassword.note')}}
+            </div>
+          </v-card>
+        </v-stepper-content>
+        <v-stepper-content step="2">
+          <v-card color="#F6F8FB">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.excelentwrk')}}
```

Comment:
one more strange word? is reader supposed to learn this language to understand first?

Reply:
this is excellent work, but a little abbreviated)

Reply:
it is really bad.